### DISCUSSION TITLE
Does milvus 2.0 able to run on a single PC?

### DISCUSSION THREAD

**Discussion by lucasjinreal** [milvus-io/milvus#26324] [Q&A and General discussion]
Does milvus 2.0 able to run on a single PC without distributed?

**Comment by xiaofan-luan** [comment]
run with standalone version should work

**Comment by lucasjinreal** [comment]
@xiaofan-luan thanks, may i ask what's the difference between milvus-lite and milvus?

Do they have big performance bias, and does they using same API?

**Reply by xiaofan-luan** [reply]
Milvus lite is only for development since it's embedded library.
Milvus is under client server mode and all request is served in dockers

**Reply by lucasjinreal** [reply]
for single machine deploy, will milvus lite caused performance issue?
